README.md

```diff
@@ -443,8 +443,8 @@ Verilog Attributes and non-standard features
 - The ``wiretype`` attribute is added by the verilog parser for wires of a
   typedef'd type to indicate the type identifier.
 
-- Various ``enum_{width}_{value}`` attributes are added to wires of an
-  enumerated type to give a map of possible enum items to their values.
+- Various ``enum_{value}`` attributes are added to wires of an enumerated type
+  to give a map of possible enum items to their values.
```

Comment:
This says the attribute name is `enum_{value}` but looking at the code it seems it actually is `enum_value_{value}`.

Reply:
Good catch, thanks!